### DISCUSSION TITLE
Importing a repository and deleting original

### DISCUSSION THREAD

**Discussion by geoscotton** [community/community#22936] [New to GitHub]
<p>Hello</p>
<p>I have gone to homepage&gt; import repository and created a new one from a private repository. If the original gets deleted, will my repository delete files aswell or it remains intact?</p>
<p>Thanks in advance.</p>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/geoscotton">@geoscotton</a>,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the How to use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Cheers!</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>If you imported <strong>anything other</strong> than a private repository from some other GitHub account, then it should be a copy of the original and deleting the original will not have any effect on your newly imported repository. If you imported a private repository from a GitHub org and it is marked as a fork of that repository, then it may still be connected and deleting the original <strong>might</strong> affect the new repository. If that is the case, you should reach out to <a href="https://github.com/contact" rel="nofollow noopener">private support</a> so they can take a look at the private account details to let you know what the situation is.</p>
